Source/SocketIO/Client/SocketIOClientSpec.swift

```diff
@@ -334,4 +334,15 @@ public enum SocketClientEvent : String {
     /// }
     /// ```
     case statusChange
+
+    /// Emitted when when upgrading the http connection to a websocket connection.
+    ///
+    /// Usage:
+    ///
+    /// ```swift
+    /// socket.on(clientEvent: .httpResponseHeaders) {data, ack in
+    ///     // Some header logic
+    /// }
+    /// ```
+    case httpResponseHeaders
```

Comment:
If this is notifying when switching transports, it should be named something `websocketUpgrade` and the payload documented.

Reply:
you're right, fixed in next PR